docs/layer_configuration.md

```diff
@@ -0,0 +1,342 @@
+- Using the Vulkan API: `VkCreateInstance()` and `VK_EXT_layer_settings`.
+- Using the `vk_layer_settings.txt` file, that can be generated by the GUI interface called *[Vulkan Configurator](https://vulkan.lunarg.com/doc/sdk/latest/windows/vkconfig.html)*.
+- Using environment variables.
+
+These three methods are implemented by the **Vulkan Layer Settings** library part of the *[Vulkan-Utility-Libraries](https://github.com/KhronosGroup/Vulkan-Utility-Libraries)* repository.
+Any layer project including this library will provide these three methods to control layer settings bringing consistency and easy of use of layers to the Vulkan community.
+
+Configuring layers means multiple tasks: Enabling layers; Ordering layers; Configuring the layers capabilities. These three aspects are described with each method to configure layers.
+
+A single setting can be configure using all three methods simulanously. A convention defines priorities regarding the different layer settings methods:
+1. The environment variable which overrides
+2. The vk_layer_settings.txt value which overrides
+3. The `VK_EXT_layer_settings` value of a layer setting.
+
+**Guideline: Settings which are unknown by the layer will be ignored independently of the method. It's the reponsability of the layer developer to ensure backward compatibility with previous versions of the layer. This is to ensure the list of layer settings remain stable across versions and that the responsability of handling layer backward compatibility doesn't fall on Vulkan application developers as this could quickly become untrackable.**.
+
+## Configuring Vulkan Layers using the *Vulkan API*
+
+### Enabling and ordering the layer using `vkCreateInstance()`
+
+Applications may programmatically activate layers via the `vkCreateInstance()` entry point. This
+is done by setting `enabledLayerCount` and `ppEnabledLayerNames` in the `VkInstanceCreateInfo`
+structure.
+
+The layer names order in `ppEnabledLayerNames` specifies the layers execution ordering from closer to the Vulkan application to closer
+to the Vulkan driver.
+
+### Configuring the layer settings using `VK_EXT_layer_settings`
+
+Layer settings may be configured using the `VK_EXT_layer_settings` extension by initializing the `VkLayerSettingsCreateInfoEXT` structure and chaining it to the `pNext` of `VkInstanceCreateInfo` when creating a Vulkan instance.
+
+### Code example
+
+```cpp
+    const char* profile_file_data = JSON_TEST_FILES_PATH "VP_LUNARG_desktop_baseline_2023.json";
+    const char* profile_name_data = "VP_LUNARG_desktop_baseline_2023";
+    const VkBool32 emulate_portability_data = VK_TRUE;
+    const char* simulate_capabilities[] = {"SIMULATE_FEATURES_BIT", "SIMULATE_PROPERTIES_BIT", "SIMULATE_EXTENSIONS_BIT"};
+
+    const VkLayerSettingEXT settings[] = {
+        {kLayerName, kLayerSettingsProfileFile, VK_LAYER_SETTING_TYPE_STRING_EXT, 1, &profile_file_data},
```

Comment:
Suggestion to provide example values for k-Constants like `kLayerName`, so the user understands it's not the constants from the library.

Reply:
Fixed